build/Microsoft.DotNet.Cli.BranchInfo.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <PropertyGroup>
     <Channel>master</Channel>
+    <CoreSetupChannel>master</CoreSetupChannel>
```

Comment:
(minor)  I'm not sure this property belongs here.  I kind of think it belongs with the other core-setup dependency properties, since they kind of go together.  This property isn't really information about the branch - instead it is information about our dependency on core-setup.